+sig/Signal.m

```diff
@@ -357,34 +357,39 @@
       b = map(a, @floor, 'floor(%s)');
     end
     
-    function a = abs(x)
+    function b = abs(a)
```

Comment:
Unless there's good reason to change the variable names, I'd prefer to keep them as Chris had them.  Often the variable names he chose convey some information about the value of the returned signal and I find them intuitive names.